main.go

```diff
@@ -25,6 +32,19 @@ func init() {
 }
 
 func main() {
-	router := routes.SetupRouter()
-	router.Run(":" + viper.GetString(`DROPKIT.PORT`))
+	// router := routes.SetupRouter()
+	_, srv := routes.SetupRouter()
+	// router.Run(":" + viper.GetString(`DROPKIT.PORT`))
+
+	shutdown := make(chan os.Signal)
+	signal.Notify(shutdown, syscall.SIGINT, syscall.SIGTERM)
+	<-shutdown
+	log.Info("Shutdown Server")
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Why did you set `5` times here?

Reply:
the 5 here means this context (a series of dependent stacks) would be closed in 5 sec

Reply:
You have to document it.